### ISSUE TITLE
quantized exmaple with CUDA Error: not a f64 F32(1e-5)

### ISSUE THREAD

**Issue by evgenyigumnov** [huggingface/candle#1912]
```
root@C.10187155:~/candle$ cargo run --example quantized --features cuda --release -- --prompt "<s> <Document 1> Invoice 234 Tomato 3 kg 400$ Nuts 4 kg 1600$ Total: 2000$ </Document 1> <Document 2> Medical card 769 Client name: John Smith History of illness: Bla Bla Bla Last visit hospital date: 17 march 1979 Personal doctor name: Anna Zuzuko </Document 2> Compare document 1 and document 2. Do these documents relate to the same area or topic? Tell me the percentage of similarity. Similarity percentage:" --which mixtral
    Finished release [optimized] target(s) in 0.40s
     Running `target/release/examples/quantized --prompt '<s> <Document 1> Invoice 234 Tomato 3 kg 400$ Nuts 4 kg 1600$ Total: 2000$ </Document 1> <Document 2> Medical card 769 Client name: John Smith History of illness: Bla Bla Bla Last visit hospital date: 17 march 1979 Personal doctor name: Anna Zuzuko </Document 2> Compare document 1 and document 2. Do these documents relate to the same area or topic? Tell me the percentage of similarity. Similarity percentage:' --which mixtral`
avx: true, neon: false, simd128: false, f16c: true
temp: 0.80 repeat-penalty: 1.10 repeat-last-n: 64
loaded 995 tensors (26.44GB) in 0.35s
Error: not a f64 F32(1e-5)
```

**Comment by LaurentMazare** [comment]
Thanks for reporting this, that's a bug I introduced yesterday, it should be fixed in #1913 .

**Comment by evgenyigumnov** [comment]
Thank you for fix! It is work now!!!
